src/osd/ReplicatedPG.cc

```diff
@@ -9893,6 +9893,8 @@ void ReplicatedPG::mark_all_unfound_lost(
       [=]() {
 	requeue_ops(waiting_for_all_missing);
 	waiting_for_all_missing.clear();
+	requeue_object_waiters(waiting_for_unreadable_object);
+	// XXX: What about waiting_for_degraded_object, waiting_for_blocked_object?
```

Comment:
@dzafman this is not addressed, we can move the missing objects from `waiting_for_blocked_object` into another map and  pass that map to `requeue_object_waiters()` instead.